pages/submission/create.tsx

```diff
@@ -1,9 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
+import { useQuery } from "react-query";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import router from "next/router";
 import { Header, TextAreaInput, TextInput } from "../../components/ui";
 import { HomeLayout } from "../../components/layout";
 import { Button } from "flowbite-react";
+import { useAuthStore } from "../../src/store/auth";
+import { createSubmission } from "../../services";
+
+export type FormSchemaType = z.infer<typeof schema>;
+
+const rMsg = "This field is required";
+
+const schema = z.object({
+  title: z.string().min(1, rMsg),
+  symptoms: z.string().min(1, rMsg),
+});
 
 const CreateSubmissionPage = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormSchemaType>({
+    resolver: zodResolver(schema),
+  });
+
+  const [values, setValues] = useState<FormSchemaType | null>(null);
+
+  useQuery(["create-submission", values], () => createSubmission(values), {
```

Comment:
wait, you're SENDING info with a useQuery? this is exactly where you want to use `useMutation` instead